Fix the references to pty(4) - should be in pty(4d) on Solaris.

This change will not be sent upstream.

--- expect5.45/libexpect.man.orig	2017-07-26 06:58:58.985845764 +0000
+++ expect5.45/libexpect.man	2017-07-26 06:59:08.395912753 +0000
@@ -670,7 +670,7 @@
 Conference, Colorado Springs, Colorado, October 17-19, 1990.
 .PP
 expect(1), alarm(3), read(2), write(2), fdopen(3), execve(2), execvp(3),
-longjmp(3), pty(4).
+longjmp(3), pty(4d).
 .PP
 There are several examples C programs in the test directory of
 .BR expect 's
